python/pip_install/tools/wheel_installer/wheel_installer.py

```diff
@@ -32,6 +33,14 @@
 from python.pip_install.tools.wheel_installer import namespace_pkgs, wheel
 
 
+@dataclass
+class Label:
+    target: str
```

Comment:
nit: "target_name" instead of just "target". A Target is a 3-tuple of (workspace, package, and name), and this is only the name portion. That, or add a comment to indicate it's only the name portion of a target.